README.md

```diff
@@ -153,10 +154,22 @@ Kilobars              | kilobars, kbar
 Pounds per Square Inch| psi
 Atmosphere            | atmosphere, at
 
+### Volume
+
+Unit                  | Names
+----------------------|---------------------------------------
+Litres                | litres, liters, L, l
+Millilitres           | millilitres, milliliters, ml, mL
+Gallons               | gallons, gal
+Quart                 | quart, qt
+Pint                  | pint, pt
+Fluid Ounces          | fluid ounces, floz
```

Comment:
I think `fl oz` is the most common abbreviation? https://en.wikipedia.org/wiki/Fluid_ounce